proposals/incubating/COMMUNITY_FUND_PROCESS/COMMUNITY_FUND_PROCESS.md

```diff
@@ -0,0 +1,42 @@
+# Community fund request, approval, and reimbursement
+
+## Infrastructure
+
+* CPC voting members create budget committee
+  * Smaller number of seats to guarantee quorum
+  * Odd number of seats to avoid deadlocks
+  * Meets monthly
+  * Elected annually
+  * Appoint an administrator for the fund.
+* New repo specific to the community fund (openjs-foundation/community-fund)
+  * Top-level README describing processes
+  * Budget file for each year containing the list of approved applicants, to be managed by the committee.
+* Mailing list for internal discussions between meetings (`cpc-private` or a new list)
+* Issue template for applications, to be determined by budget committee
+* Labels for application stages, can only be changed by budget committee
+  * Under consideration
+  * Need more information
+  * Declined
+  * Accepted
+* LFX Crowdfunding used to disburse the payments, once approved.
+
+## Eligibility requirements
+
+* Must not have a source of corporate funding for the activity.
+* Request must be directly related to one or more OpenJS Foundation projects or initiatives.
+* Must apply >1 month before the proposed use of funds, and provide an itemized breakdown of anticipated expenses.
+* Must provide writeup for openjsf.org blog summarizing use of funds to be eligible for future funding.
+
+## Application process
+
+1. Applicants open an issue in the community-fund repo
```

Comment:
All the list items seem to have their index numbers set to 1 in this list and the list below. Should these even be ordered lists?

Reply:
That's a standard markdown thing. The generated markdown will use the correct numbers, but if you add or remove things, you don't have to change all the numbers in the list yourself.